spec/models/filters/criteria/filter_for_gender_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative 'shared_filter_criteria_context'
+
+RSpec.describe Filters::Criteria::FilterForGender do
+  include_context 'filter criteria setup'
+
+  let(:genders) { [0, 1] } # Female and Male
+  let(:filter) { ::Filters::FilterBase.new(user_id: user.id, genders: genders) }
+  let(:criteria) { described_class.new(input: filter, config: config) }
+
+  let!(:enrollments) do
+    [
+      create_enrollment_for_client(create(:hud_client, Gender: 0, Woman: 1)),
+      create_enrollment_for_client(create(:hud_client, Gender: 1, Man: 1)),
+      create_enrollment_for_client(create(:hud_client, Gender: 4, NonBinary: 1)),
```

Comment:
I'm probably missing something, but why are we passing `Gender` here?  That field was removed with the FY2024 updates.

Reply:
this was slop, removed it. Thanks!